run.docker

```diff
@@ -0,0 +1,27 @@
+# run.docker
+FROM deployment-base
+
+ADD . /srv/hondana-git
+
+COPY nginx.conf.in /etc/nginx/sites-enabled/hondana
+
+RUN apt-get install -qy \
+		-o APT::Install-Recommends=false -o APT::Install-Suggests=false \
+		nginx supervisor && \
+	apt-get clean && \
+	unlink /etc/nginx/sites-enabled/default && \
+	sed -e "s|HONDANA_SOURCES|/srv/hondana-git|g" -i /etc/nginx/sites-enabled/hondana && \
+	sed -e "s|HONDANA_STORE|/srv/store|g" -i /etc/nginx/sites-enabled/hondana && \
+	/srv/env/bin/pip install --no-index -f /srv/hondana-git/wheelhouse hondana gunicorn && \
+	/srv/env/bin/python \
+		/srv/hondana-git/scripts/generate_conf.py \
+		/srv/hondana-git/config.yaml.j2 /srv/hondana-git/config.yaml
```

Comment:
Yes, I would use volumes. Environment variables are often proposed in peoples' best practice suggestions, but I am not such a fan.

If you prefer, an alternative is to deploy a config template and have an `ENTRYPOINT` that is a script to template that from env vars. See the official postgres container for an entrypoint example.